liquibase-core/src/main/java/liquibase/datatype/core/DoubleType.java

```diff
@@ -44,7 +44,7 @@ public DatabaseDataType toDatabaseDataType(Database database) {
         }
 
         if (database instanceof OracleDatabase) {
-            return new DatabaseDataType("FLOAT", 24);
+            return new DatabaseDataType("DOUBLE PRECISION");
```

Comment:
Can we combine all if conditions for various databases to one single if with || ?

Reply:
Good suggestion, I pushed an extra change for this